src/main/steps/applicant1/payment-callback/get.ts

```diff
@@ -47,7 +48,9 @@ export default class PaymentCallbackGetController {
 
     req.session.save(() => {
       if (payments.wasLastPaymentSuccessful) {
-        return res.redirect(APPLICATION_SUBMITTED);
+        return req.session.userCase.applicationType === ApplicationType.JOINT_APPLICATION
```

Comment:
I wonder if it would be better to have a function in the urls file that takes in the applicationType and returns the correct url, instead of repeating the same check every time